docs/tutorials/platform_beginner/6_enabling_account_registration.md

```diff
@@ -0,0 +1,71 @@
+```markdown
+
+```
+
+# Step 6 - Enabling account registration
+
+We are trying to let our friends write their bike trips directly into our website. We would like to let them create Rides. As an Administrator of the website you could create the Users yourself but In order to let them create their own content, first, you want to let them create their own accounts.
+
+## Add the registration possible
+
+From the Admin Panel, go to the _Roles_ management screen, click on _Anonymous_ Role name link to edit it.
+
+![step_6_role_mgmt_screen](/home/shaim/www/developer-documentation/docs/tutorials/platform_beginner/img/step_6_role_mgmt_screen.png)
```

Comment:
Link here and others in the page point to local files

Reply:
OMG: I let it! Thank you!